rotkehlchen/globaldb/cache.py

```diff
@@ -69,7 +69,7 @@ def globaldb_get_general_cache_values_like(
 ) -> list[str]:
     """Function to read globaldb cache. Returns all the values that start with key."""
     cache_key = _compute_cache_key(key_parts)
-    cursor.execute('SELECT value FROM general_cache WHERE key=?', (cache_key,))
+    cursor.execute('SELECT value FROM general_cache WHERE key LIKE ?', (f'{cache_key}%',))
```

Comment:
What are you doing here? Why this change? Reading the name and the docstring it sounds correct, but why was it equality before?

Reply:
I suppose it was an error in the implementation. Maybe copy-paste. This function is not used in the code atm, so it was not affecting anything

Reply:
If it's not used at all whatsoever then remove it. It's a pleasure to remove un-needed things. If we nedit we add something like that.